src/python/pants/backend/jvm/targets/jvm_binary.py

```diff
@@ -167,7 +167,9 @@ def skip_signatures_and_duplicates_concat_well_known_metadata(cls, default_dup_a
     :returns: JarRules
     """
     default_dup_action = Duplicate.validate_action(default_dup_action or Duplicate.SKIP)
-    additional_rules = assert_list(additional_rules, expected_type=(Duplicate, Skip))
+    additional_rules = assert_list(additional_rules,
+                                   expected_type=(Duplicate, Skip),
+                                   field_name='additional_rules')
```

Comment:
I think we need to leave the field name off on this one.  Adding the field name 'additional_rules' won't make any sense to the user.